.githooks/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Don't push unless the local branch contains the head of the remote
+# origin/develop branch.
+
+set -e
+
+# Get the head of the branch being pushed.
+#
+# Local SHAs are passed to stdin, in the second column. See pre-push.sample
+read _ local_sha _ _ 
+
+# Make sure we have the latest version of develop, locally
+git remote update > /dev/null
+
+# Check that pushee branch contains develop head SHA
+remote_develop_head=`git rev-list origin/develop | head -1`
+if [ -z "`git log $local_sha | grep $remote_develop_head | head -1`" ] ; then
+    echo "↘↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↙"
+    echo "→ Please rebase on origin/develop before pushing ←"
```

Comment:
Loving the arrows :+1:

Reply:
Yeah, I'm worried it'll get lost in the rest of the git push output.